pack/Makefile

```diff
@@ -2,6 +2,8 @@ all: package
 
 PACKDIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 
+VERSION := $(shell sed 's|\-\([alpha|beta|rc]\)|~\1|' <<< $(VERSION))
```

Comment:
`<<<` does not work on dash.

Reply:
Regarding the regex:

1. The dash does not require backslash escaping.
2. The square brackets catch any symbol listed inside them.
3. You use the pipe symbol as for delimiter as well as inside the regex.

It seems, the correct sed expression would be `'s@-\(alpha\|beta\|rc\)@~\1@'`.

Reply:
Is not config.mk more appropriate place for that?

Reply:
Maybe it would be better to transform a version within deb.mk, rpm.mk, because the tilda syntax is the package manager specific thing.

It worth to add a section into readme and describe how such alpha/beta/rc version will be transformed.

Reply:
I agree with you, this change seems wrong.